debian/changelog

```diff
@@ -1,3 +1,9 @@
+rspamd (1.6.1) unstable; urgency=medium
+
+  * New release.
+
+ -- Sebastian Lipponer <mail@sebastianlipponer.de>  Sat, 24 Jun 2017 22:12:07 +0200
```

Comment:
Which script in particular are you referring to? Nevertheless, then just apply the other commit to add the missing build-dependencies.